src/lib/jsonParseV2/parseEntity.cpp

```diff
@@ -151,6 +151,5 @@ std::string parseEntity(ConnectionInfo* ciP, Entity* eP, bool eidInURL)
     }
   }
 
-  eP->present("");
```

Comment:
Given that you are fixing the pressent() issue, I would suggest to do it completely. In that sense, have a look to https://github.com/telefonicaid/fiware-orion/pull/1379 and remove also the other present() line and include the same CNR comment about it.